builder/azure/common/devicelogin.go

```diff
@@ -40,8 +41,11 @@ var (
 
 // Authenticate fetches a token from the local file cache or initiates a consent
 // flow and waits for token to be obtained.
-func Authenticate(env azure.Environment, tenantID string, say func(string)) (*adal.ServicePrincipalToken, error) {
+func Authenticate(env azure.Environment, tenantID string, say func(string), apiScope string) (*adal.ServicePrincipalToken, error) {
```

Comment:
let's call `apiScope` just `scope`, since that is the right oauth term.

Reply:
fixed